ui/Theme.svelte

```diff
@@ -1,3 +1,9 @@
+<!-- Copyright © 2021 The Radicle Upstream Contributors
+  --
+  -- This file is part of radicle-upstream, distributed under the GPLv3
+  -- with Radicle Linking Exception. For full terms see the included
+  -- LICENSE file.
+  -->
```

Comment:
Do svelte files not have a line break after the licence on purpose? .ts files seem to have a line break.

Reply:
Good spot. Prettier will remove the additional line break so we can’t add it.